projects/plugins/vaultpress/readme.txt

```diff
@@ -7,7 +7,7 @@ Stable tag: 2.2.0
 Requires PHP: 5.6
 License: GPLv2
 
-Jetpack VaultPress offers real-time backups, one-click restores, and premium support from WordPress experts.
+(DEPRECATED: Please install "Jetpack VaultPress Backup" instead) Jetpack VaultPress offers real-time backups, one-click restores, and premium support from WordPress experts.
```

Comment:
I would suggest using a link pointing to https://wordpress.org/plugins/jetpack-backup/.
Also, I think the idea discussed in peaFOp-sK-p2#comment-319 is to use this message as the short description (removing the other part), could you double-check?

```suggestion
(DEPRECATED: Please install [Jetpack VaultPress Backup](https://wordpress.org/plugins/jetpack-backup/) instead) Jetpack VaultPress offers real-time backups, one-click restores, and premium support from WordPress experts.
```

Reply:
> I would suggest using a link pointing to https://wordpress.org/plugins/jetpack-backup/.

Some fierce googling has revealed that the short description can't have markup:

https://wordpress.org/plugins/readme.txt

> Also, I think the idea discussed in peaFOp-sK-p2#comment-319 is to use this message as the short description (removing the other part), could you double-check?

For now, maybe we'll leave the "(depreciation notice) A short description" form (even if it's more than 150 characters) and see how it goes.